test/distributed/org/apache/cassandra/distributed/test/MigrationCoordinatorTest.java

```diff
@@ -116,7 +119,6 @@ public void explicitVersionIgnore() throws Throwable
             IInstanceConfig config = cluster.newInstanceConfig();
             config.set("auto_bootstrap", true);
             IGNORED_SCHEMA_CHECK_VERSIONS.setString(initialVersion.toString() + ',' + oldVersion);
-            System.setProperty("cassandra.consistent.rangemovement", "false");
```

Comment:
Why was this one removed?

Reply:
Because it is already set in the node initialization, see https://github.com/apache/cassandra/pull/2972/files#diff-321b52220c5bd0aaadf275a845143eb208c889c2696ba0d48a5fc880551131d8L596